project/AppDependencies.scala

```diff
@@ -6,30 +6,33 @@ object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-26"  % "5.3.0",
-    "uk.gov.hmrc" %% "domain"                      % "5.11.0-play-26",
-    "uk.gov.hmrc" %% "http-caching-client"         % "9.5.0-play-26",
+    "com.typesafe.akka" %% "akka-protobuf" % "2.6.14",
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-28"  % "5.12.0",
+    "uk.gov.hmrc" %% "domain"                      % "6.2.0-play-28",
+    "uk.gov.hmrc" %% "http-caching-client"         % "9.5.0-play-28",
     "uk.gov.hmrc" %% "tax-year"                    % "1.1.0",
-    "uk.gov.hmrc" %% "play-partials"               % "8.1.0-play-26",
+    "uk.gov.hmrc" %% "play-partials"               % "8.2.0-play-28",
     "uk.gov.hmrc" %% "emailaddress"                % "3.5.0",
-    "uk.gov.hmrc" %% "govuk-template"              % "5.66.0-play-26",
-    "uk.gov.hmrc" %% "play-ui"                     % "9.4.0-play-26",
-    "com.typesafe.play" %% "play-json-joda"        % "2.7.4",
+    "uk.gov.hmrc" %% "govuk-template"              % "5.69.0-play-28",
+    "uk.gov.hmrc" %% "play-ui"                     % "9.6.0-play-28",
+    "com.typesafe.play" %% "play-json-joda"        % "2.8.1",
     "com.typesafe.play" %% "play-iteratees"        % "2.6.1",
-    compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.4" cross CrossVersion.full),
-    "com.github.ghik" % "silencer-lib" % "1.7.4" % Provided cross CrossVersion.full
+    "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.12.3",
+    compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.5" cross CrossVersion.full),
+    "com.github.ghik" % "silencer-lib" % "1.7.5" % Provided cross CrossVersion.full
   )
 
   val test: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"             %% "hmrctest"           % "3.10.0-play-26",
-    "org.scalatest"           %% "scalatest"          % "3.0.9",
-    "org.scalatestplus.play"  %% "scalatestplus-play" % "3.1.3",
+    "org.scalatest"           %% "scalatest"          % "3.2.9",
+    "org.scalatestplus.play"  %% "scalatestplus-play" % "5.1.0",
+    "org.scalatestplus"      %% "scalatestplus-mockito"   % "1.0.0-M2",
     "org.pegdown"             %  "pegdown"            % "1.6.0",
-    "org.jsoup"               %  "jsoup"              % "1.13.1",
+    "org.jsoup"               %  "jsoup"              % "1.14.2",
     "com.typesafe.play"       %% "play-test"          % PlayVersion.current,
     "org.mockito"             %  "mockito-core"       % "1.10.19",
     "com.github.tomakehurst"  %  "wiremock-jre8"      % "2.28.0",
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-26" % "5.3.0"
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "5.12.0",
```

Comment:
i think you can remove this `bootstrap-frontend-play-28` from here

Reply:
done